### DISCUSSION TITLE
Validate dynamic select-boxes

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#816] [PrimeVue]
I am posting same question second time, hope this time I will get the solution.
So the code below is a fragment from official doc. which validates one select-box:
`
<template>
    <div class="card flex justify-content-center">
        <form @submit="onSubmit" class="flex flex-column align-items-center gap-2">
            <MultiSelect v-model="value" :class="{ 'p-invalid': errorMessage }" :options="cities" optionLabel="name" placeholder="Select Cities" :maxSelectedLabels="3" class="w-full md:w-20rem" aria-describedby="text-error" />
            <small id="text-error" class="p-error">{{ errorMessage || '&nbsp;' }}</small>
            <Button type="submit" label="Submit" />
        </form>
        <Toast />
    </div>
</template>

<script setup>
import { useToast } from 'primevue/usetoast';
import { useField, useForm } from 'vee-validate';

const { handleSubmit, resetForm } = useForm();
const { value, errorMessage } = useField('value', validateField);
const toast = useToast();

const cities = [
    { name: 'New York', code: 'NY' },
    { name: 'Rome', code: 'RM' },
    { name: 'London', code: 'LDN' },
    { name: 'Istanbul', code: 'IST' },
    { name: 'Paris', code: 'PRS' }
];

function validateField(value) {
    if (!value) {
        return 'City is required.';
    }

    return true;
}

const onSubmit = handleSubmit((values) => {
    let detail = '';

    values.value.map((value, i) => {
        detail += value.name + (i < values.value.length - 1 ? ', ' : '');
    });

    if (values.value && values.value.length > 0) {
        toast.add({ severity: 'info', summary: 'Form Submitted', detail: detail, life: 3000 });
        resetForm();
    }
});
</script>
`

how can I validate dynamic select-boxes? for example suppose I have below code which generates dynamic multiple select-boxes:
`<PrimeSelect
               v-for="filter in carsFilters"
               v-model="selectedFilters[filter.name]"
               filter
               @change="selectFilter"
               :id="filter.name"
               :options="[filter]"
               optionGroupLabel="name"
               optionGroupChildren="values"
               :placeholder="filter.name"
               :selectionLimit="1"
               class="w-full sm:w-19rem mt-3"
               panelClass="z-1 select-none"
           />`

**Comment by tugcekucukoglu** [comment]
Stackblitz demo will make it much easier.

**Reply by Uzeyir9695** [reply]
So, here is a Stackblitz link: [Validate Form](https://stackblitz.com/edit/k6svjz?file=src%2FApp.vue).  Please move to this link and help me to validate that form.

**Comment by tugcekucukoglu** [comment]
I have tried to change `selectedCities` as `value` and it seems to be working. I am not sure how exactly VeeValidate naming working.

**Reply by Uzeyir9695** [reply]
could you please modify to display error messages? I couldn't, it didn't work.
